mandatory/1-fix-functions.js

```diff
@@ -10,7 +10,7 @@
 
 */
 
-function getMood() {
+function getMood(isHappy) {
```

Comment:
while you realized that you need to add the isHappy as a parameter there is still the case of variable shadowing that blocks this function to behave properly

Reply:
still trying to understand these differences and their behaviour. however i believe i fixed it now  👍